### PR TITLE
tea: New package

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

File: mingw-w64-tea/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.9.2
+pkgrel=1
+pkgdesc='A command line tool to interact with Gitea servers (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+msys2_references=(
+  'archlinux: tea'
+)
+msys2_repository_url='https://gitea.com/gitea/tea'
```
Comment (Biswa96) [inline mingw-w64-tea/PKGBUILD:15]: I think it is not required to add `msys2_repository_url` when the value is same as `url` variable.

Comment (podsvirov) [reply mingw-w64-tea/PKGBUILD:15]: Your rigth! Ok. Will fixed.

**Comment by podsvirov** [comment]
I copy winpty wrapper (tea.sh) from github-cli but it's realy needed? As for me, wrapper only broke output::

```
$ tea.exe --version
Version: 0.9.2  golang: 1.21.5
```
but
```
$ tea --version
Version: ←[1m0.9.2←[0m  golang: 1.21.5
```
I plan to remove the wraper.

File: mingw-w64-tea/PKGBUILD (hunk 1 above)
Comment (podsvirov) [reply mingw-w64-tea/PKGBUILD]: Done.

**Pull request merged**